src/models/wallet/SimpleWallet.ts

```diff
@@ -93,6 +93,16 @@ export class SimpleWallet extends Wallet {
     );
   }
 
+  /**
+   * Create a SimpleWallet from object
+   * @param wallet - wallet object from outside source
+   * @returns {SimpleWallet}
+   */
+  public static createFromObject(wallet: SimpleWallet): SimpleWallet {
```

Comment:
I'm not sure i understand what the word "Object" means in createFromObject. Is there a better wording?